src/baal/ensemble.py

```diff
@@ -0,0 +1,73 @@
+from copy import deepcopy
+from typing import Dict, List
+
+import torch
+from torch import nn
+
+from baal import ModelWrapper
+from baal.modelwrapper import _stack_preds
+from baal.utils.cuda_utils import to_cuda
+
+
+class EnsembleModelWrapper(ModelWrapper):
+    """
+    Wrapper that support ensembles with the same architecture.
+
```

Comment:
Add: If you're looking to use ensembles for non-deep models, see our sklearn tutorial: baal.readthedocs.io/en/latest/notebooks/sklearn_tutorial.html